cmd/k8sruntime_util.go

```diff
@@ -199,6 +199,16 @@ func getContainers(resource Resource) (container []ContainerV1) {
 	return container
 }
 
+// Get PodSpec from the PodV1 resource type to check for PSC
+
+func getPodSpecs(resource Resource) (podSpec PodSpecV1) {
+	switch kubeType := resource.(type) {
+	case *PodV1:
```

Comment:
do we need to use switch when there is only one resource type ?

Reply:
switch is used to by choice to maintain code style, also taking into consideration that we may add more resources to kubeaudit, also using type switch results in much cleaner code, if we're to use a if statement in this case, we'd be calling multiple functions on the resource to get the type. Also taking into account #127, I'd suggest that we use a switch for now

Reply:
ohh okay I see.

Reply:
ohhh I see

Reply:
A type switch is the right thing here to do.